.lintrunner.toml

```diff
@@ -1251,6 +1251,7 @@ exclude_patterns = [
     'test/test_testing.py',
     'test/test_torch.py',
     'test/test_transformers.py',
+    'test/xpu/test_transformers.py',
```

Comment:
Why is this excluded?

Reply:
The `xpu/test_transformers.py` meant to contain a selection of tests in `test_transformers.py`. As the `test_transformers.py` is excluded from PYFMT, it will be difficult to compare between the 2 tests if the lint is included for `xpu/test_transformers.py`.